symphony-bdk-core/src/test/java/com/symphony/bdk/core/auth/AuthenticatorFactoryTest.java

```diff
@@ -64,6 +64,30 @@ void testGetBotAuthenticatorWithValidPrivateKey(@TempDir Path tempDir) throws Au
     assertEquals(BotAuthenticatorRsaImpl.class, botAuth.getClass());
   }
 
+  @Test
+  void testGetAuthenticatorWithValidPrivateKeyContent() throws AuthInitializationException {
+
+    final BdkConfig config = new BdkConfig();
+    config.getBot().setPrivateKeyContent(RSA_PRIVATE_KEY.getBytes());
+
+    final AuthenticatorFactory factory = new AuthenticatorFactory(config, DUMMY_API_CLIENT_FACTORY);
+    final BotAuthenticator botAuth = factory.getBotAuthenticator();
+
+    assertNotNull(botAuth);
```

Comment:
not needed IMHO as we use it just the line after

Reply:
+1